### PR TITLE
add workshop info

### PR DESCRIPTION
**Pull request by cldougl**

### CODE REVIEW

File: tutorial/run.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def create_contents(contents):
 
             html.Div(className="links", children=[
                 html.A('pricing', className="link", href="https://plot.ly/products/on-premise"),
+                html.A('workshop', className="link", href="https://plotcon.plot.ly/workshops"),
```
Comment (chriddyp) [inline tutorial/run.py]: Maybe plural? `workshops`?

Comment (chriddyp) [reply tutorial/run.py]: that way it's not like interpreted as some type of weird product (like "Workspace")

Comment (cldougl) [reply tutorial/run.py]: sure, since we only had one now I put it singular but hopefully there will be more to come!

File: tutorial/support.py [hunk 2]
```diff
@@ -15,6 +15,15 @@
...
 a demo of Dash and Dash On-Premises too.
 
+### Dash Workshop
+
+We will be hosting a two day Dash workshop at
+[Plotcon NYC](https://plotcon.plot.ly/) this November!
+The workshop will start with the foundations and guiding principles of Dash
+and move into advanced Dash practices with plenty of time for individual
+questions. You can register for the workshop and see more information at:
+https://plotcon.plot.ly/workshops.
```
Comment (chriddyp) [inline tutorial/support.py]: [https://plotcon.plot.ly/workshops](https://plotcon.plot.ly/workshops)

**Comment by chriddyp** [comment]
💃  once you consider my comments

**Pull request merged**